remote_test.go

```diff
@@ -1210,6 +1212,41 @@ func (s *RemoteSuite) TestList(c *C) {
 	}
 }
 
+func (s *RemoteSuite) TestListPeeling(c *C) {
+	remote := NewRemote(memory.NewStorage(), &config.RemoteConfig{
+		Name: DefaultRemoteName,
+		URLs: []string{"https://github.com/aide/aide.git"},
```

Comment:
This has the potential of becoming abandoned and/or flaky.

Reply:
Figured out how to run the test, see https://github.com/go-git/go-git/pull/750/files#r1183171371

Reply:
Thank you for the review. Replaced the repository with the suggestion. PTAL